.circleci/config.yml

```diff
@@ -7,11 +7,22 @@ version: 2.1
 orbs:
   maven: circleci/maven@1.0.3
 
+machine: true
+
 jobs:
   run_checkstyle:
     <<: *defaults
     working_directory: /tmp/repos/cmo-metadb
+    environment:
+      - TESTCONTAINERS_RYUK_DISABLED: true
     steps:
+      - setup_remote_docker:
+          version: 20.10.2
+      - run:
+          name: "Configure testcontainers.properties..."
+          command: |
+            touch /home/circleci/.testcontainers.properties && \
+            echo "DOCKER_HOST = unix:///var/run/docker.sock" > /home/circleci/.testcontainers.properties
```

Comment:
just trying things out here. `testcontainers.properties`, if we actually need it, can be checked into `src/test/resources`

Reply:
This should normally not be necessary and Testcontainers should detect the Docker environment by itself.